lib/script/script.js

```diff
@@ -1131,7 +1131,7 @@ class Script {
             || !(sig[sig.length - 1] & Script.hashType.SIGHASH_FORKID))
             subscript.findAndDelete(sig);
 
-          validateTXSignature(sig, flags);
+          CheckTransactionSignature(sig, flags);
```

Comment:
Function names are generally camel cased with the first letter being lowercase in this codebase. There are multiple functions that are camel cased that start with uppercased letters